### PR TITLE
adding HaveAnnotation rule

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**
This PR should solve #229

### CODE REVIEW

File: src/Analyzer/ClassDescription.php [hunk 1]
```diff
@@ -141,4 +146,18 @@ public function isAbstract(): bool
     {
         return $this->abstract;
     }
+
+    public function getAnnotation(): string
```
Comment (fain182) [inline src/Analyzer/ClassDescription.php:150]: this is a container of multiple annotations? maybe this should be plural to be more coherent with hasAnnotation?

Comment (AlessandroMinoccheri) [reply src/Analyzer/ClassDescription.php:150]: yes, I agree, I fix it immediately.

**Review by fain182 — APPROVED** [review]
:+1:

**Review by micheleorselli — COMMENTED** [review]
I do like the idea behind this PR, but I am not sure I fully understood the scope. We are saying we want to check if a cIass has a specific annotation? 

If is that so, I am wondering if the name `Annotation` could be misleading since we are checking if a DocBlock contains a string, so for instance if we have this snippet of code

```php

/**
* This is just a comment
*/
class Banana {}
```

the rule `new HaveAnnotation('just a')` will be matched. Usually, when we say "annotation" we think about something like this

```php
/**
* @annotation 
*/
```

**Comment by AlessandroMinoccheri** [comment]
Good point @micheleorselli could it be better `new HaveDocBlock('just a')` or is there a better name?

**Comment by micheleorselli** [comment]
> Good point @micheleorselli could it be better `new HaveDocBlock('just a')` or is there a better name?

DocBlockContains?
HaveDocBlockContaining?

**Review by micheleorselli — APPROVED** [review]

**Pull request merged**